MidCamp/midcamp
# Fix composer.json, add enable Webform module, build a survey, build a sign up form

# Description
- Fixes duplicate key from #125 (added a Core patch for Views datetime fields)
- Adds Webform, exports basic config
- Builds out a MidCamp survey
- Builds out two entityreference Views to control selection of Speakers, Sessions in the webform
- Creates webform to sign up for Lightning Talks

# To test
- Run `composer install`.  Observe that the duplicate key message is gone.
- Import config with `drush cim -y`.  Observe that Webform is enabled.
- Visit http://midcamp.org.docker.amazee.io/form/midcamp-2018-survey.  Observe the survey exists.
- Look at the View responsible for returning actual Speakers and not just anybody who is a site user at http://midcamp.org.docker.amazee.io/admin/structure/views/view/user_entityreference/edit/entity_reference_1.  
  - Notice that it is using relationships for `field_people` and `field_schedule_location` to grab only users who are on scheduled Sessionss.  
  - Validate that this is behaving as expected
- Look at the View responsible for returning accepted Sessions at http://midcamp.org.docker.amazee.io/admin/structure/views/view/survey_filters.  Validate this is working properly and not returning every Session that exists.
- Visit http://midcamp.org.docker.amazee.io/topic/lightning-talks.  Observe there is a sign up form here.

# Screenshots
Composer Error before:
![image](https://user-images.githubusercontent.com/4048700/36899346-401b280e-1de4-11e8-955c-1178a8d70c7b.png)

## New survey example:
![image](https://user-images.githubusercontent.com/4048700/36903148-6f8edf3c-1df2-11e8-8b3f-3297096b2b4f.png)

## Lightning Talk Webform
![midcamp org docker amazee io_topic_lightning-talks](https://user-images.githubusercontent.com/4048700/37069676-070a17ca-217a-11e8-9ca2-4c7c5dd87571.png)
